napari_allencell_annotator/widgets/template_list.py

```diff
@@ -109,6 +110,11 @@ def add_item(self, name: str, key: Key | ComboKey):
                 for opt in key.get_options():
                     widget.addItem(opt)
             widget.setCurrentText(default)
+        else:
+            annot_type = ItemType.POINT
+            widget = QPushButton("Select")
+            widget.setFixedWidth(200)
```

Comment:
I dont think styling should go here. unless there's a good reason to do so. You can apply a style when using addwidget() instead

Reply:
moved in https://github.com/aics-int/napari-allencell-annotator/pull/92/commits/f0a9cde427560fa25e875b67b0d7e3576c4edc5a